main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Dog
+//  action 1: The dog barks.
+dog.bark();
+//  action 2: The dog sleeps.
+dog.sleep();
+//  action 3: The dog eats.
+dog.eat();
 
 //  2)
+//  Noun: Radio
+//  action 1: The radio receives signals.
+radio.receiveSignal();
+//  action 2: The radio transmits signals.
+radio.transmitSignal();
+//  action 3: The radio changes frequencies.
+radio.changeFrequency();
+
+//  3) musicPlayer
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  action 1: The music player plays a song.
+musicPlayer.play();
+//  action 2: The music player stops playing a song.
+musicPlayer.stop();
+//  action 3: The music player opens a new file.
+musicPlayer.openFile();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Car 
+//  action 1: The car accelerates.
+car.accelerate();
+//  action 2: The car slows down.
+car.slowDown();
+//  action 3: The car changes gear. 
+car.changeGear();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Bird
+//  action 1: The bird flys.
+bird.fly();
+//  action 2: The bird sings.
+bird.sing();
+//  action 3: The bird lays an egg.
+bird.layEgg();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Student
+//  action 1: The student reads.
+student.read();
+//  action 2: The student takes notes.
+student.takeNotes();
+//  action 3: The student takes exam.
+student.takeExam();
+
+//  7) 
+//  Noun: alarmClock
+//  action 1: The alarm clock sounds alarm. 
+alarmClock.alarmSound();
+//  action 2: The alarm clock sleeps alarm. 
+alarmClock.alarmSleep();
+//  action 3: The alarm clock disables alarm.  
+alarmClock.alarmDisable();

```

Comment:
the action word in a function name should be the first word in the name
Also, using`alarm` is redundant in the function name; the name of the object already says `alarm` in it
```
alarmClock.sound();
alarmClock.sleep();
alarmClock.disable();
```
- [ ] revise